content/guides/code-splitting-import.md

```diff
@@ -144,6 +144,28 @@ module.exports = {
 };
 ```
 
+## `import` imports the entire module namespace
+
+Consider the following two examples:
+
+```js
+// Example 1: top-level import
+import * as Component from './component';
+// Example 2: Code-splitting with import()
+import('./component').then(Component => /* ... */);
+```
```

Comment:
Maybe show a three liner of `component` exporting `default`+`foo`+`bar` to illustrate the context.